### ISSUE TITLE
Неправильно склоняется «Айн-Темушент»

### ISSUE THREAD

**Issue by temi4** [morpher-ru/uservoice#26]
Делал через API 

Айн-Темушент - он склонил еще и слово Айн, получилось "в Айне-Темушенте"
По идее айн не нужно склонять?

**Comment by temi4** [comment]
https://disk.yandex.ru/d/Qemc3Hzvm1_7GA - все области, где скорей всего не правильно сосклоняло. 
Правда тут области из других стран мира.

**Comment by temi4** [comment]
https://disk.yandex.ru/d/yuOjNQ73kW4YNQ - примеры районов областей из РФ, которые плохо склоняет

**Comment by bzaar** [comment]
Да, много спорных моментов, но много и полезного. Спасибо вам за это
исследование!

**Comment by bzaar** [comment]
Исправил: https://ws3.morpher.ru/russian/declension?s=Айн-Темушент

Для списков завел отдельные тикеты.
